### PR TITLE
add version flag

### PR DESCRIPTION
**Pull request by theverything**

### CODE REVIEW

**Comment by vjeux** [comment]
Thanks, I've wanted this for the browser version!

**Comment by jlongster** [comment]
@vjeux Won't you need it from the API instead of the CLI for the browser?

**Comment by vjeux** [comment]
Yeah, from the API indeed!

**Comment by theverything** [comment]
@vjeux @jlongster Not sure if this is what you meant by adding it to the api.

File: index.js [hunk 1]
```diff
@@ -74,5 +75,6 @@ function formatWithShebang(text, opts) {
 module.exports = {
   format: function(text, opts) {
     return formatWithShebang(text, opts);
-  }
+  },
+  version: version
```
Comment (vjeux) [inline index.js]: <3

**Comment by jlongster** [comment]
Nice, yep that's perfect! I would like this as well because on my blog post I show the version and right now it's hardcoded.

**Pull request merged**